applications/luci-app-uhttpd/root/usr/share/rpcd/acl.d/luci-app-uhttpd.json

```diff
@@ -2,10 +2,17 @@
 	"luci-app-uhttpd": {
 		"description": "Grant UCI access for luci-app-uhttpd",
 		"read": {
-			"uci": [ "uhttpd" ]
+			"uci": [ "uhttpd" ],
+			"file": {
+				"/*": ["read"]
+			}
 		},
 		"write": {
-			"uci": [ "uhttpd" ]
+			"uci": [ "uhttpd" ],
+			"file": {
+				"/*": ["write"],
```

Comment:
Damn, I missed that line.  @jow-  That means that you can write all the files! I Have to revert @Ayushmanwebdeveloper your changes

Reply:
@aparcar I think we need to revert or fix this!

Reply:
Hi!, Actually, I granted that permission because the old app also allowed the user to remove certificate or key file from anywhere in the system. I first set it to luci-uploads.

Reply:
First I thought that luci-uploads or at max  /etc would be the best because system keep cert and key in /etc. But then I thought to keep it the same as the old app.

Reply:
One option would be do add a luci.uhttpd rpcd to remove the file form the uci config.
```
uhttpd.main.cert
uhttpd.main.key
```
So we do not need to grande access to the whole file system.

I am thinking here of ACL, which is also possible with the LuCI. I don't want the user to be able to change everything.

Reply:
So I think it will suffice and be okay to restrict write access to only /etc/luci-uploads.

Reply:
Sounds good

Reply:
I fixed it here https://github.com/openwrt/luci/pull/6438